<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>onlinePKGame</class>
 <widget class="QMainWindow" name="onlinePKGame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1100</width>
    <height>660</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton:hover
{
	background-color: rgb(226, 127, 40);
	border: none;
	color: #FFF;
	border-radius: 3px;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="sendButton">
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>570</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Agency FB</family>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>发送</string>
    </property>
   </widget>
   <widget class="QPushButton" name="failButton">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>570</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Agency FB</family>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>认输</string>
    </property>
   </widget>
   <widget class="QPushButton" name="readyButton">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>570</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Agency FB</family>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>游戏准备</string>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>100</y>
      <width>371</width>
      <height>451</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QTableWidget" name="userTableWidget">
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">background: rgb(255, 147, 15);</string>
       </property>
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <column>
        <property name="text">
         <string>主机名</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>用户名</string>
        </property>
        <property name="background">
         <color>
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </property>
       </column>
       <column>
        <property name="text">
         <string>IP地址</string>
        </property>
        <property name="background">
         <color>
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </property>
       </column>
      </widget>
     </item>
     <item>
      <widget class="QTextBrowser" name="messageBrowser">
       <property name="styleSheet">
        <string notr="true">background: rgb(255, 147, 15);</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTextEdit" name="messageTextEdit">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>50</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>50</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1100</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_A">
    <property name="title">
     <string>开始（&amp;A）</string>
    </property>
    <widget class="QMenu" name="menu">
     <property name="title">
      <string>新的一局</string>
     </property>
     <property name="icon">
      <iconset resource="res.qrc">
       <normaloff>:/image/new.png</normaloff>:/image/new.png</iconset>
     </property>
     <addaction name="playAction"/>
     <addaction name="computerAction"/>
    </widget>
    <addaction name="menu"/>
    <addaction name="backAction"/>
    <addaction name="exitAction"/>
   </widget>
   <widget class="QMenu" name="menu_P">
    <property name="title">
     <string>游戏（&amp;P）</string>
    </property>
    <addaction name="saveAction"/>
   </widget>
   <addaction name="menu_A"/>
   <addaction name="menu_P"/>
  </widget>
  <action name="backAction">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/image/home.png</normaloff>:/image/home.png</iconset>
   </property>
   <property name="text">
    <string>返回主菜单</string>
   </property>
  </action>
  <action name="exitAction">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/image/quit.png</normaloff>:/image/quit.png</iconset>
   </property>
   <property name="text">
    <string>退出游戏</string>
   </property>
  </action>
  <action name="saveAction">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/image/screenshot.png</normaloff>:/image/screenshot.png</iconset>
   </property>
   <property name="text">
    <string>保存棋局</string>
   </property>
  </action>
  <action name="playAction">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/image/person.png</normaloff>:/image/person.png</iconset>
   </property>
   <property name="text">
    <string>玩家PK</string>
   </property>
  </action>
  <action name="computerAction">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/image/computer.png</normaloff>:/image/computer.png</iconset>
   </property>
   <property name="text">
    <string>人机PK</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
